libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -602,3 +608,94 @@ $gd-header-project-width: 230px;
         }
     }
 }
+
+.gd-data-header-menu-section {
+    display: flex;
+    flex: 1 0 auto;
+
+    .icon-sources {
+        border-bottom-right-radius: 0;
+        border-top-right-radius: 0;
+
+        &:hover {
+            border-color: $button-border-hover;
+        }
+    }
+
+    .icon-model {
+        border-radius: 0;
+
+        &:hover {
+            border-color: $button-border-hover;
+        }
+    }
+
+    .icon-cloud-upload {
+        border-bottom-left-radius: 0;
+        border-top-left-radius: 0;
+
+        &::before {
+            top: 11px;
+            color: #b0beca;
+        }
+
+        &:hover {
+            border-color: $button-border-hover;
+        }
+    }
+
+    button.is-active[class*="gd-button-"]::before,
+    button[class*="gd-button-"]::before {
+        width: 19px;
+        height: 16px;
+        font-size: 16px;
+    }
+
+    .gd-button-primary[class*="icon-"] {
+        box-shadow: none;
+        padding-left: 35px;
```

Comment:
Is this the padding between the upload icon and the left edge of the button? If that's right, I think the value should be `11px` as used in the mockup.

Reply:
Changed it thanks